### ISSUE TITLE
Clipped a web page with YouTube linked videos - they auto-play!

### ISSUE THREAD

**Issue by LorenAmelang** [zadam/trilium#1449]
45.4

[Glutes log.txt](https://github.com/zadam/trilium/files/5578946/Glutes.log.txt)

I clipped a web page that has lots of inline YouTube videos: 
https://www.drbodygadget.com/hip-flexor-stretches/
(Other pages I clipped tonight with videos also auto-play...) 

When I open the note, or even point to it in search results, most of the videos begin playing - simultaneously! Audio chaos! 

I can scroll to each one and pause it, but the next time the note is opened or refreshed they all play again - from the beginning. They are not child notes, only the still images are children. If I _Edit_ the note they all stop, in fact they are invisible. Each one is an empty Paragraph. I have no idea where the links are stored. 

There are a couple that don't auto-play. I can't remember for sure, but I suspect it is the ones I watched before clipping the page that auto-play. I know YouTube keeps the status of every video you watch, and applies it across apps and even different machines. If I return to that page in my browser, I get the same simultaneous auto-play. From (I think) the same point in each video. But the other pages do the same thing, so I'm blaming YouTube...  

I'm not sure Trilium can do much about this...  Maybe it could include the videos only as visible URLs we could click on to see the videos? I suspect if it actually links to the YouTube URLs, trying to get the initial image to display, YouTube will just begin playing them. The link content would probably let us recognize which one we want to see. Or maybe there is a way to tell YouTube to not auto-play a link?

**Comment by zadam** [comment]
Hi, is this happening in the read-only mode, i.e. is there this button:

![](https://user-images.githubusercontent.com/617641/99915683-63f4b500-2d05-11eb-9943-114172b60831.png)

In either case this is quite surprising - I'm actually not even able to start the youtube videos when I click on them (which is a known issue in CKEditor ...).

**Comment by LorenAmelang** [comment]
Yes, when they auto-play, that edit button is visible. If I click Edit, the videos stop - and disappear from the edit view. 

Except when I mouse over a global search result that references the note. Then I get the rounded-corner popup preview, with no edit button but all the audio playing. 

Just did a global search for "youtube". It looks like Trilium used to save just an attribute, like:
#clipType=page #pageUrl="https://www.youtube.com/watch?v=O_GEUdxQETA&list=PLANsIt1t8GkE2t-zbyYA7sXoWjRlUrXu9&index=5" 

But those were clipped directly from YouTube itself. These new ones are embedded in a third-party web page. They do not auto-play there...  Their youtube URL does not appear in global search. 

Just tried a new clip: 
https://animoto.com/blog/video-marketing/embed-video-on-website

The embedded YouTube videos do not appear in Trilium, whether I've watched and paused them or not. In Edit mode, there is no blank paragraph where they should appear. 

Strange!
